x/observer/client/cli/tx_update_operational_flags.go

```diff
@@ -0,0 +1,59 @@
+package cli
+
+import (
+	"encoding/json"
+	"os"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/spf13/cobra"
+
+	"github.com/zeta-chain/node/x/observer/types"
+)
+
+func CmdUpdateOperationalFlags() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update-operational-flags",
+		Short: "Broadcast message UpdateOperationalFlags",
+		Args:  cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, _ []string) (err error) {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			var operationalFlags types.OperationalFlags
+
+			flagSet := cmd.Flags()
+			file, _ := flagSet.GetString("file")
+			restartHeight, _ := flagSet.GetInt64("restart-height")
```

Comment:
Why not just use a command line argument and accept a filepath . 
That would be similar to the logic we have for all other queries.

The values should be declared as constants if you need to use a flag. Although I would still be in favour of using the args as it keep the codebase uniform

Reply:
> Why not just use a command line argument and accept a filepath .

I wanted to make interactive testing easier.

> The values should be declared as constants if you need to use a flag.

Yes